substratevm/src/com.oracle.svm.core.graal.llvm/src/com/oracle/svm/core/graal/llvm/LLVMGenerator.java

```diff
@@ -283,8 +283,9 @@ LLVMValueRef getFunction(ResolvedJavaMethod method) {
     }
 
     byte[] getBitcode() {
-        assert builder.verifyBitcode();
         byte[] bitcode = builder.getBitcode();
+        assert builder.verifyBitcode();
```

Comment:
This contradicts your comment, the module should be verified before the bitcode is generated.

Reply:
Good catch.